app/jobs/resolution_proofing_job.rb

```diff
@@ -38,6 +38,7 @@ def perform(
 
     user = User.find_by(id: user_id)
 
+    document_capture_session = DocumentCaptureSession.new(result_id: result_id)
```

Comment:
Moved this up so that we can pass it to the ProgressiveProofer constructor to use as the discriminator for the A/B test.